lib/iris/fileformats/grib/message.py

```diff
@@ -60,7 +60,7 @@ def messages_from_filename(filename):
                     break
                 raw_message = _RawGribMessage(grib_id)
                 recreate_raw = _MessageLocation(filename, offset)
-                yield _GribMessage(raw_message, recreate_raw)
+                yield GribMessage(raw_message, recreate_raw)
 
     def __init__(self, raw_message, recreate_raw):
         """
```

Comment:
The docs don't explain the meaning of 'recreate_raw' at all.
I think that needs fixing now we're making it public.